### ISSUE TITLE
Switch between satellite view and normal view

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#189]
## User story
1. As a User
2. I want a UI element
3. So that I can switch between to views (satellite and normal)

## Acceptance criteria
* Button or Slider to switch between views

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Lucas-Nan** [comment]
![Image](https://github.com/amosproj/amos2024ss04-building-information-enhancer/assets/166795065/4deffb87-c347-4322-a934-9dc1d9bc68cf)



![Image](https://github.com/amosproj/amos2024ss04-building-information-enhancer/assets/166795065/c3a78519-757f-4850-9a3a-66a93fb91b1a)

bottom right has btns - visual improvement should be done with non transparent background
